extension/src/websocket-forwarder.ts

```diff
@@ -33,6 +35,22 @@ export class WebSocketForwarder implements vscode.Disposable {
     });
   }
 
+  private handleClientMessage(message: string) {
+    const obj = JSON.parse(message);
```

Comment:
same way as glsp does it?
Don't get me wrong, nothing wrong here, but maybe JSON.parse can be a performance leak here, as it is (as far as I know) not ultimative performant (but maybe also only with really big data, which at least in this direction should never be a problem)